### ISSUE TITLE
Failed to activate the atom-hg package: e.getOriginURL is not a function

### ISSUE THREAD

**Issue by victor-torres** [victor-torres/atom-hg#55]
I have the same issue. I installed and reinstall but it does not works.

**Atom**: 1.34.0 x64
**Electron**: 2.0.16
**OS**: Microsoft Windows 10 Pro
**Thrown From**: [atom-hg](https://github.com/victor-torres/atom-hg) package 2.0.14

### Stack Trace

Failed to activate the atom-hg package

```
At e.getOriginURL is not a function

TypeError: e.getOriginURL is not a function
    at atom.project.observeRepositories.subscriptions.add.atom.project.observeRepositories.e (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:212429)
    at Function.simpleDispatch (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:1172318)
    at Emitter.emit (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:1173759)
    at Project.addPath (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:410818)
    at Project.setPaths (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:409619)
    at e.consume.e (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:412718)
    at Provider.t.exports.Provider.provide (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:14:1029640)
    at ServiceHub.t.exports.ServiceHub.provide (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:3012661)
    at Package.activateServices (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:3020847)
    at Package.activateNow (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:3017902)
    at measure (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:3017226)
    at Package.measure (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:3014857)
    at activationPromise.activationPromise.Promise (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:3017086)
    at new Promise (<anonymous>)
    at Package.activate (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:3017029)
    at PackageManager.activatePackage (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:382643)
    at ~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:899185
    at r (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:11:893791)
    at o (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:14:1032547)
    at ChildProcess.n.process.on.e (~/AppData/Local/atom/app-1.34.0/resources/app/static/<embedded>:14:1032753)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
```

### Commands

```
```

### Non-Core Packages

```
atom-hg 2.0.14 
busy-signal 2.0.0 
chestnut-light-atom-syntax 0.2.1 
file-icons 2.1.27 
intentions 1.1.5 
language-cfml 0.23.0 
linter 2.2.0 
linter-cflint 0.15.0 
linter-ui-default 1.7.1 
minimap 4.29.9 
oceanic-next 1.0.0 
```

_Originally posted by @albertcito in https://github.com/victor-torres/atom-hg/issues/47#issuecomment-452705128_

**Comment by victor-torres** [comment]
This is an exception caused by the [atom/metrics](https://github.com/atom/metrics) package introduced by the version [1.6.2](https://github.com/atom/metrics/releases/tag/v1.6.2).

Apparently it does not interfer with the atom-hg plugin functionality, but I'm gonna fix this providing the required API.

**Comment by victor-torres** [comment]
@albertcito it should be fixed in version 2.0.15. Could you update and confirm it, please?

**Comment by albertcito** [comment]
Now it's working well. 

But I don't know if is my computer only but it shows the following error (In any case it does not affect the work with hg):

```
Uncaught (in promise) TypeError: Cannot read property 'indexOf' of undefined
    at Repository.handleHgError (file:///C:/Users/albert\./.atom/packages/atom-hg/lib/hg-utils.coffee:301:18)
    at file:///C:/Users/albert\./.atom/packages/atom-hg/lib/hg-utils.coffee:342:8
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```
